src/path_converters.h

```diff
@@ -305,7 +309,11 @@ class PathClipper : public EmbeddedQueue<3>
         : m_source(&source),
           m_do_clipping(do_clipping),
           m_cliprect(rect),
+          m_lastX(0.0),
```

Comment:
It might be better to initialize these to `nan`?  0 is a valid value, but they should not have a valid value until the first time we set them.

Reply:
ok